core/src/migrations/000094-exportDestinationAllowNull.ts

```diff
@@ -0,0 +1,20 @@
+import Sequelize, { DataTypes } from "sequelize";
+
+export default {
+  up: async (queryInterface: Sequelize.QueryInterface) => {
+    await queryInterface.changeColumn("exports", "destinationId", {
+      type: DataTypes.STRING(191),
+      allowNull: true,
+    });
+  },
+
+  down: async (
+    queryInterface: Sequelize.QueryInterface,
+    DataTypes: typeof Sequelize
+  ) => {
+    await queryInterface.changeColumn("exports", "destinationId", {
+      type: DataTypes.STRING(191),
+      allowNull: false,
+    });
+  },
+};
```

Comment:
I'm not sure if this migration should also `null` out destinaitonIds for exports who's destinationId does not exist anymore. It would match the behavior going forward, but the sweepers should still take care of them eventually

Reply:
👍 This is a "going-forward" fix.